specification/personal-demographics.yaml

```diff
@@ -302,9 +305,6 @@ paths:
     get:
       summary: Search for a patient
       operationId: search-patient
-      externalDocs:
-        description: ODS codes and documentation.
-        url: https://fhir.nhs.uk/Id/ods-organization-code
```

Comment:
I cannot see where this was move to? Was it moved to a file outside of this repo?

Reply:
That should not have been removed - let me add it back in.

Reply:
Correction: I had moved it to the main page rather than that specific endpoint - making sure that change is pushed now.

Reply:
Which page are you referring to as the main page? Is it in a different repo?